scylladb/scylladb.go

```diff
@@ -137,7 +137,8 @@ func (s *Storage) Get(key string) ([]byte, error) {
 		}
 		return nil, err
 	}
-	if expiration > 0 && time.Now().Unix() > expiration {
+	// If the expiration time has already passed, then return nil
+	if expiration != 0 && expiration <= time.Now().Unix() {
```

Comment:
Does this mean that the entry stays in the DB? Maybe we should call delete instead once it's expired

Reply:
Great Point, not sure how I missed that, will update shortly.

Reply:
Updated to use default scylladb TTL